src/commands/utils/wolframalpha.ts

```diff
@@ -1,82 +1,112 @@
-                                    podTexts.push(`**${pod.title}:** \n${results.join('\n')}`);
-                                }
-                            }
-                        });
-                        if (podTexts.length > 0) {
-                            const result = podTexts.join('\n\n');
-                            const queryParams = new URLSearchParams({ i: query });
-                            // await msg.channel.send(result + ' - [Web Result](' + WOLFRAMALPHA_QUERY_URL + queryParams.toString() + ')');
 
-                            const waEmbed = makeEmbed({
-                                description: makeLines([
-                                    result,
-                                    '',
-                                    `[Web Result](${WOLFRAMALPHA_QUERY_URL}${queryParams.toString()})`,
-                                ]),
-                            });
-
-                            await msg.reply({ embeds: [waEmbed] });
-                        } else {
-                            await msg.reply('Wolfram Alpha did not give an answer.');
+            if (response.queryresult.success === true) {
+                const podTexts: string[] = [];
+                response.queryresult.pods.forEach((pod) => {
+                    if (pod.id !== 'Input' && pod.primary === true) {
+                        const results: string[] = [];
+                        pod.subpods.forEach((subpod) => {
+                            results.push(subpod.plaintext);
+                        });
+                        if (results.length > 0) {
+                            podTexts.push(`**${pod.title}:** \n${results.join('\n')}`);
                         }
-                    } else {
-                        await msg.reply('Wolfram Alpha could not understand your query.');
                     }
+                });
+                if (podTexts.length > 0) {
+                    const result = podTexts.join('\n\n');
+                    const queryParams = new URLSearchParams({ i: query });
+
+                    const waEmbed = makeEmbed({
+                        description: makeLines([
+                            result,
+                            '',
+                            `[Web Result](${WOLFRAMALPHA_QUERY_URL}${queryParams.toString()})`,
+                        ]),
+                    });
+
+                    await msg.reply({ embeds: [waEmbed] });
+                    return;
                 }
+                const noResultsEmbed = makeEmbed({
+                    title: 'Wolfram Alpha Error | No Results',
+                    description: makeLines([
+                        'No results were found for your query.',
+                    ]),
+                    color: 'RED',
+                });
+                await msg.channel.send({ embeds: [noResultsEmbed] });
+                return;
+            }
+            await msg.reply('Wolfram Alpha could not understand your query.');
```

Comment:
See here - https://discord.com/channels/738864299392630914/897491699167793182/998342460738633799

One embed missed. Can you fix in another PR :)